assets/xml/objects/object_nb.xml

```diff
@@ -1,46 +1,59 @@
 ﻿<Root>
+    <!-- Assets for Anju's Grandma -->
     <File Name="object_nb" Segment="6">
-        <Animation Name="object_nb_Anim_000290" Offset="0x290" />
-        <Animation Name="object_nb_Anim_00052C" Offset="0x52C" />
-        <Animation Name="object_nb_Anim_0006D4" Offset="0x6D4" />
-        <Animation Name="object_nb_Anim_000990" Offset="0x990" />
-        <DList Name="object_nb_DL_004A70" Offset="0x4A70" />
-        <DList Name="object_nb_DL_005170" Offset="0x5170" />
-        <DList Name="object_nb_DL_005430" Offset="0x5430" />
-        <DList Name="object_nb_DL_0056F0" Offset="0x56F0" />
-        <DList Name="object_nb_DL_005988" Offset="0x5988" />
-        <DList Name="object_nb_DL_005AA8" Offset="0x5AA8" />
-        <DList Name="object_nb_DL_006390" Offset="0x6390" />
+        <!-- Anju's Grandma Animations-->
+        <Animation Name="gNbTalkAnim" Offset="0x290" />
+        <Animation Name="gNbAngryAnim" Offset="0x52C" />
+        <Animation Name="gNbRelievedAnim" Offset="0x6D4" />
+        <Animation Name="gNbIdleAnim" Offset="0x990" />
+
+        <!-- Anju's Grandma DisplayLists -->
+        <!-- Includes legs -->
+        <DList Name="gNbWheelchairDL" Offset="0x4A70" />
```

Comment:
Guessing this applies to the first DL, so probably should go to the right of it rather than above.